### PR TITLE
Update azure_rm_sqlfirewallrule related document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_sqlfirewallrule related document.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_sqlfirewallrule/azure_rm_sqlfirewallrule_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule.py [hunk 1]
```diff
@@ -36,24 +36,25 @@
...
         description:
-            - The start IP address of the firewall rule. Must be IPv4 format. Use value C(0.0.0.0) to represent all Azure-internal IP addresses.
+            - The start IP address of the firewall rule.
+            - Must be IPv4 format. Use value C(0.0.0.0) to represent all Azure-internal IP addresses.
     end_ip_address:
         description:
-            - "The end IP address of the firewall rule. Must be IPv4 format. Must be greater than or equal to startIpAddress. Use value C(0.0.0.0) to represe
-               nt all Azure-internal IP addresses."
+            - The end IP address of the firewall rule.
+            - Must be IPv4 format. Must be greater than or equal to startIpAddress. Use value C(0.0.0.0) to represent all Azure-internal IP addresses.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule.py]:
```suggestion
            - Must be IPv4 format. Must be greater than or equal to I(start_ip_address). Use value C(0.0.0.0) to represent all Azure-internal IP addresses.
```

File: lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule.py [hunk 2]
```diff
@@ -36,24 +36,25 @@
...
     state:
-      description:
-        - Assert the state of the SQL Database. Use C(present) to create or update an SQL Database and C(absent) to delete it.
-      default: present
-      choices:
-        - absent
-        - present
+        description:
+            - Assert the state of the SQL Database. Use C(present) to create or update an SQL Database and C(absent) to delete it.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_sqlfirewallrule.py]:
```suggestion
            - State of the SQL Database. Use C(present) to create or update an SQL Database and C(absent) to delete it.
```

**Review by acozine — COMMENTED** [review]
Overall looks good - I see a couple of small ways to make it even better. Thanks @Fred-sun for going through all of these modules!

**Comment by Fred-sun** [comment]
@acozine Thanks for your review. Update new.

**Comment by Fred-sun** [comment]
ready_for_review

**Pull request merged**